locales/uk/translation.json

```diff
@@ -365,17 +365,17 @@
     "show-header-navigation": "Вказівник заголовків",
     "show-tab-search": "Відображати форму пошуку по поточній вкладці",
     "open-verse-lists-in-new-tab": "Перелік віршів у новій вкладці",
-    "open-in-new-tab": "Open in new tab",
+    "open-in-new-tab": "Відкрити в новій вкладці",
     "show-search-results-in-popup": "Відображати результати пошуку у вириначі",
     "show-notes-tags-indicators": "Позначки нотаток та міт",
     "show-tags": "Міти",
     "use-tag-group-filter": "Використати цідило мітових груп",
     "show-notes": "Нотатки",
-    "hide-empty-verse-notes": "Hide empty verse notes",
+    "hide-empty-verse-notes": "Приховати порожні нотатки до віршів",
     "limit-notes-display-size": "Обмежити висоту нотаток",
     "limit-text-width": "Обмежити ширину тексту Біблії",
-    "use-tags-column": "Міти у окремому стовпці",
-    "use-notes-column": "Use notes column",
+    "use-tags-column": "Міти в окремому стовпці",
+    "use-notes-column": "Нотатки в окремому стовпці",
```

Comment:
@tobias-klein, it chose the preposition similar to the line 377, but it's actually better to use the other one stylistically (it was incorrect in line 377). Grammatically everything was right